cmd/imagec/imagec.go

```diff
@@ -694,18 +694,22 @@ func main() {
 		options.token = token
 	}
 
-	// Get the manifest
-	manifest, err := FetchImageManifest(options)
-	if err != nil {
-		log.Fatalf("Error while pulling image manifest: %s", err)
-	}
-
 	// HACK: Required to learn the name of the vmdk from given reference
 	// Used by docker personality until metadata support lands
 	if !options.resolv {
 		progress.Message(po, options.tag, "Pulling from "+options.image)
 	}
 
+	// Get the manifest
+	manifest, err := FetchImageManifest(options)
+	if err != nil {
+		if strings.Contains(err.Error(), "image not found") {
```

Comment:
Thanks for your input! I figured I didn't have the image name in `fetcher.go`, that's why I propagate `"image not found"` and add the name in `imagec.go`.

I believe the image name would be in the `url` struct there, so let me see if I can parse it out.